### DISCUSSION TITLE
connect to wallet in loop

### DISCUSSION THREAD

**Discussion by BianKo666niglio** [Chia-Network/chia-blockchain#7027] [Support]
Hi, I have download the v.0.0.4, installed the 0.0.2simple and change some ports (other forks have the same ports).
when I try to open the Gui is in "connect to the wallet" loop.

any tricks?

**Comment by loppefaaret** [comment]
are you sure you are in the right place ? this is chia-blockchain - we are currently on release 1.1.7 here

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
